infra/k8s/greptime/eks-deployment-runbook.md

```
@@ -21,7 +23,18 @@ Replace the `<PLACEHOLDER>` values in `cluster.yaml` with your VPC, subnet, and
 eksctl create cluster -f infra/k8s/greptime/cluster.yaml
 ```
 
-## 2) Create S3 bucket (object storage)
+## 2) Install the AWS Load Balancer Controller
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)